Makefile

```diff
@@ -120,23 +114,11 @@ LIBMCOUNT_TARGETS += libmcount/libmcount-single.so libmcount/libmcount-fast-sing
 
 _TARGETS := uftrace libtraceevent/libtraceevent.a
 _TARGETS += $(LIBMCOUNT_TARGETS) libmcount/libmcount-nop.so
-_TARGETS += misc/demangler misc/symbols
```

Comment:
This removes them from default make target so it changes the behavior.  Please keep it unchanged.

Reply:
@namhyung it can be same effect by adding "@ $ (MAKE) -C $ (srcdir) / misc" under 'all:' command. like below:

Reply:
Why not adding `misc` to `TARGETS`?  I don't know it's good to have separate Makefile.

Reply:
@namhyung This is because each time you add a program to misc you need to add the relevant script to the makefile. I want to add a new program to misc, but before that I want to clean up the Makefile.

Reply:
Anyway you still need to change misc/Makefile.